examples/standard/params.json

```diff
@@ -12,5 +12,6 @@
         "tenant_name": "myaadtenant.onmicrosoft.com",
         "stack_config_path": "../examples/standard/stack/stack_config.json",
         "users_groups_path": "../examples/standard/users_and_groups.json",
-        "log_level": "info"
-}
+        "log_level": "info",
+        "databricks_resource_id": '2ff814a6-3304-4ab8-85cb-cd0e6f879c1d'
```

Comment:
Can we avoid adding a credential unless its a redacted example

Reply:
done, removed comments

Reply:
this is databricks resource id, we can remove it, its not needed anymore